kms/apiv1/key_management_client.go

```diff
@@ -2966,7 +2967,7 @@ func (c *keyManagementRESTClient) UpdateCryptoKey(ctx context.Context, req *kmsp
 		if err != nil {
 			return nil, err
 		}
-		params.Add("updateMask", string(updateMask))
+		params.Add("updateMask", strings.Replace(string(updateMask), "\"", ""))
```

Comment:
Should be replaceAll

Reply:
Thanks didn't mean to commit this, hald way "ctrl+z" mistake